src/ca_biositing/pipeline/ca_biositing/pipeline/utils/reviewed_api_mappings.py

```
@@ -30,20 +57,1356 @@
+    'EQUINE': 'EQUINE',
+    'FIGS': 'FIGS',
+    'FLAXSEED': 'FLAXSEED',
+    'GARLIC': 'GARLIC',
+    'GINGER ROOT': 'GINGER ROOT',
+    'GOATS': 'GOATS',
+    'GUAVAS': 'GUAVAS',
+    'HAY  ALL (DRY)': 'HAY',
+    'HAY  OTHER (DRY)': 'HAY',
+    'HAZELNUTS': 'HAZELNUTS',
+    'HOGS': 'HOGS',
+    'HOGS': 'HOGS',
```

Ruff (0.15.5)

[error] Dictionary key literal `'HOGS'` repeated

Remove repeated key literal `'HOGS'`

(F601)